presto-main/src/main/java/com/facebook/presto/operator/scalar/Centroid.java

```diff
@@ -27,130 +27,165 @@
 /**
  * A single centroid which represents a number of data points.
  */
-public class Centroid implements Comparable<Centroid>, Serializable {
+public class Centroid
+        implements Comparable<Centroid>, Serializable
+{
     private static final AtomicInteger uniqueCount = new AtomicInteger(1);
 
-    private double centroid = 0;
-    private int count = 0;
+    private double mean;
+    private int count;
 
     // The ID is transient because it must be unique within a given JVM. A new
     // ID should be generated from uniqueCount when a Centroid is deserialized.
     private transient int id;
 
-    private List<Double> actualData = null;
+    private List<Double> actualData;
 
-    private Centroid(boolean record) {
+    private Centroid(boolean record)
+    {
         id = uniqueCount.getAndIncrement();
         if (record) {
             actualData = new ArrayList<>();
         }
     }
 
-    public Centroid(double x) {
+    public Centroid(double value)
+    {
         this(false);
-        start(x, 1, uniqueCount.getAndIncrement());
+        start(value, 1, uniqueCount.getAndIncrement());
     }
 
-    public Centroid(double x, int w) {
+    public Centroid(double value, int weight)
+    {
         this(false);
-        start(x, w, uniqueCount.getAndIncrement());
+        start(value, weight, uniqueCount.getAndIncrement());
     }
 
-    public Centroid(double x, int w, int id) {
+    public Centroid(double value, int weight, int id)
+    {
         this(false);
-        start(x, w, id);
+        start(value, weight, id);
     }
 
-    public Centroid(double x, int id, boolean record) {
+    public Centroid(double value, int id, boolean record)
+    {
         this(record);
-        start(x, 1, id);
+        start(value, 1, id);
     }
 
-    Centroid(double x, int w, List<Double> data) {
-        this(x, w);
+    Centroid(double value, int weight, List<Double> data)
+    {
+        this(value, weight);
         actualData = data;
     }
 
-    private void start(double x, int w, int id) {
+    private void start(double value, int weight, int id)
+    {
         this.id = id;
-        add(x, w);
+        add(value, weight);
     }
 
-    public void add(double x, int w) {
+    public void add(double value, int weight)
+    {
         if (actualData != null) {
-            actualData.add(x);
+            actualData.add(value);
         }
-        count += w;
-        centroid += w * (x - centroid) / count;
+        count += weight;
+        mean += weight * (value - mean) / count;
     }
 
-    public double mean() {
-        return centroid;
+    public double mean()
```

Comment:
If we truly adhere to Presto style, should these be `getMean`, `getCount` and `getId` instead?